### ISSUE TITLE
Can't unbind LB+Y on controllers(xbox) to "Open WoodHarvesterControls menu"

### ISSUE THREAD

**Issue by BainrowKiss** [BargonMods/FS22_WoodHarvesterControls#2]
Pressing LB+Y opens the wood harvester controls settings even though there's no input bound in the settings. If I bind for example LB+Dpad Down, I get the both prompts in the drop-down menu.
Am I not supposed to use LB+Y to change cutting length, and instead just rely on length presets and the bucking system? 

Either way, thanks for a great mod! Shame Giants couldn't add something as simple as dropping a tree for the processor head. 
Thought I'd only be using the drop function so I was a bit overwhelmed at first with all the options, but I'm glad I decided to go for your mod: Bucking system looks interesting, grabbing on to trees and even adjusting the height while grabbed on before cutting, feeding the tree forth and back after cutting, there's just so much here I didn't know I wanted! Great work! Will recommend it to everyone who'll listen.

**Comment by BargonMods** [comment]
First of all, thanks for your feedback!

The WoodHarvesterControls mod overrides the base-game stuff so there is no "change length" binding. You have the menu instead. 
So yeah, the "Bucking System" is now the place to change your standard length (or lengths if you want to use more than one and let the computer choose depending on the diameter).
The "Length Presets" is an extra feature that let you "force" a specific length with a button.

I'm working now to improve the user experience changing some options, the default values, the order in the menu and also the names (current names are maybe too specific or advance)

**Comment by BainrowKiss** [comment]
Awesome, thanks for the explanation! 
I've seen the "vehicle control add-on" mod has added several sections to the "help menu" with some info and explanations of different settings. If you know how to, I think that would make a great addition for your mod. 

Personally I think the names are good enough, since there's a detailed description next to all the settings. 
I don't know about the values or ranges, I only noticed the cut length was a bit different from the default vanilla ones. Do you work with these machines in real life? I think the default bucking length was set to 370, so I was thinking perhaps the trucks you usually deliver logs to irl want them to be this length, or did you just pick some numbers at random? If you ever make a section in the help menu, some info about the most realistic settings would be nice. Perhaps even some info about the best way to cut down a forest and sorting logs or how to safely cut down trees in a steep hill.

This is definitely on my top three mod list now, I doubt I'll ever play FS without it, thanks again!

**Comment by BargonMods** [comment]
Yes, it's a very good idea, I'll try to do it!

I have more experience with forwarders than harvesters, but I know how they work and also talked a lot with different operators to find out how different brands work etc. 
Those lengths are from an irl price matrix from Sweden. It depends a lot on the sawmill or pulp mill and it also changes between countries, so those are just some defaults that can obviously be changed (to an exact number you enter instead of the predefined lengths from the base game script). I think this will be a perfect match for the new Platinum DLC production chains so you can make different lengths for different diameters and deliver the different assortments to the different factories.

When I release the final version I will create a detailed guide on my YouTube channel explaining everything in detail, how things work, how to configure the controls and settings to make it work like the different irl harvesters and also some tips and tricks to operate the machine properly. 

But yes, adding information to the help menu is definitely a good idea!

**Comment by BainrowKiss** [comment]
Been using this mod last few days, and what a game changer.. I absolutely love it! Been using the experimental mode quite a bit, and starting to get the hang of it. The added realism and control of the harvester head, the bucking system, they're all amazing, so much more control of everything. 10/10 mod.. 

If you want some more feedback, here's a few more suggestions and observations (I'm hoping that 1 & 2 might be possible to change without too much work or effort from you, but I have no idea, so I'm absolutely fine with it being as it is. 10/10 !! :

1. Don't know if you missed this in my initial post; The button combo to open the harvester menu can't be rebound or changed, so if it can be done, I think it would be nice to free up this button (LB+Y) on the controller to use for other things, since there's not that much use for the menu after the final adjustments have been made. 
2. I think it would be cool when after you've fed a tree to computer length, if you press the button again it feeds it to the 2nd bucking system length, and then back to the first length if you press it a third time and are only using two assortments, or the third length if you're using three. (Think I saw this detail when watching "how to operate a forest machine" by Skogsforum.se on yt) :) 
3. After cutting or picking up a tree, if I feed it backwards until there's no wood where the saw is cutting and hit cut, it will shake around frantically before dropping the tree. Easily avoidable by the player by having a bit bigger margin for the bottom cut after feeding.

A few other "buggy" behaviors I've found; when I'm grabbing onto trees, it seems like in tensions up the the boom and the machine, so when I cut it, everything shakes around a bit as the machine settles. I think the limited control when using a gamepad makes it more difficult to find the resting position of the boom/machine after grabbing onto a tree. That, and sometimes when grabbing after cutting, I think the lower part of the harvester is grabbing onto the root part and the already cut tree at the same time causing some weird behavior. -My fix for this is to just cut the tree, move the boom up a tiny inch, grab onto the tree, "tilt" the harvester head and then move the boom so that the tree falls where I want it. I'm going to get myself some dual joysticks :D I started using this mod cause I wanted to be able to drop the tree I'm holding, so when these bugs happened it was easy get out of by just dropping the tree. Vanilla games throws the machine around way more..

_Edit: Remapped the whole controller so that I'm simulating dual joysticks on the xbox controller, and there's a lot less tension in the boom after grabbing. Now I just grab and cut and it works great!_

Looks fun and challenging working with these machines irl, it's cool how they seem to float above the terrain when moving.. Ended up watching harvester video I mentioned and "Skota ännu smartare" av Skogforsk on yt the other day.. I learned quite a lot, specially the forwarding bit. I only wish for some better physics (and sounds) in the game after watching the skogforsk video..lol . I play a lot on the Rennebu map; you really get the sense of logging up in the mountain sides of Norway and it reminds me so much of the valley I live in. From what I could tell from a few searches online, we're operating with the same lengths in the price matrix in Norway as you do, so I feel lucky that whatever information you'll put in will probably feel like close to home.. :)
Looking forward to the Platinum release, a must have DLC for loggers I guess.. Your mod is as well :) It's going to a be a perfect match I'm sure! 

Sorry for the lengthy comment, don't feel any need or rush to answer back. Looking forward to see what changes and improvements you will bring to this nearly perfect mod! Thanks again for sharing this mod with us.
